### ISSUE TITLE
CPacketClickWindow.usedButton rename

### ISSUE THREAD

**Issue by gabizou** [ModCoderPack/MCPBot-Issues#256]
So, the way the `usedButton` (field_149553_c) is used isn't actually finding the used button but a bit masked combination of the used button and various other masked values. Some of them are more evident than others, but none the less, I believe are somewhat self documented when you look at how [Sponge interprets the "clickMode"](https://github.com/SpongePowered/SpongeCommon/blob/4e1817a28adec2c6dd20e4f55480e6ec02f92634/src/main/java/org/spongepowered/common/event/tracking/phase/PacketPhase.java#L108-L147) (new [link](https://github.com/SpongePowered/SpongeCommon/blob/bleeding/src/main/java/org/spongepowered/common/event/tracking/phase/packet/PacketPhase.java#L183-L227)) and [how it unpacks the information](https://github.com/SpongePowered/SpongeCommon/blob/4e1817a28adec2c6dd20e4f55480e6ec02f92634/src/main/java/org/spongepowered/common/event/tracking/phase/PacketPhase.java#L366-L371) (new [link](https://github.com/SpongePowered/SpongeCommon/blob/bleeding/src/main/java/org/spongepowered/common/event/tracking/phase/packet/PacketPhase.java#L349-L359)).

So, in short, the field itself is likely an old name, but now should be renamed, especially that the use of it isn't matching the parameter assigned for `Container.slotClick(int slotId, int dragType, ClickType clickTypeIn, EntityPlayer player)`

**Comment by bspkrs** [comment]
please update the dead links so I can take a look.

**Comment by bspkrs** [comment]
Do you have a suggestion for how it should be named?

**Comment by gabizou** [comment]
I was thinking of `packedClickData` or something of sorts.

**Comment by bspkrs** [comment]
mind setting the comment on that?
done. `packedClickData` it is.
